ar-001/auth.json

```diff
@@ -0,0 +1,283 @@
+            "code_input_error_incorrect_code": "الرجاء إدخال رمز تأكيد صحيح",
+            "code_input_error_expired_code": "لقد انتهت صلاحية رمز التأكيد الخاص بك، يرجى طلب رمز جديد",
+            "didnt_receive_a_code_description": "لم تتلق رمز؟",
+            "didnt_receive_a_code_link": "إعادة إرسال الرمز",
+            "disclaimer": "من خلال المتابعة، فإنك توافق على سياساتنا",
+            "disclaimer_terms_of_use_link": "الشروط",
+            "disclaimer_privacy_link": "سياسة الخصوصية"
+        },
+        "email_address_verification_email": {
+            "email_body": "ستنتهي صلاحية هذا الرمز خلال ساعتين.  إذا كنت تعتقد أنه لا ينبغي أن تتلقى هذه الرسالة الإلكترونية، فيمكنك تجاهلها بأمان.",
+            "email_preview": "الرجاء استخدام الرمز أدناه لتأكيد عنوان بريدك الإلكتروني والمتابعة في ${business_name}",
+            "email_subject": "رمز التحقق من البريد الإلكتروني",
+            "email_greeting": "أهلاً",
+            "email_disclaimer": "لقد تلقيت هذا البريد الإلكتروني لأنك طلبت رمز التأكيد من ${business_name}"
+        },
+        "password_setup_page": {
+            "page_title": "تعيين كلمة المرور | ${business_name}",
+            "heading": "إعداد كلمة المرور",
+            "description": "يرجى تعيين كلمة المرور الخاصة بك",
+            "password_input_label": "كلمة المرور",
+            "password_input_error_extra_space": "لا يمكن أن تبدأ كلمة المرور أو تنتهي بمسافة",
+            "password_input_error_too_common": "هذه كلمة المرور شائعة جدًا. اجعلها أكثر صعوبة في التخمين.",
+            "password_input_error_too_short": "يجب أن تتكون كلمات المرور من 8 أحرف على الأقل",
+            "confirm_password_input_label": "تأكيد كلمة المرور",
+            "confirm_password_input_error_mismatch": "تأكد من أن كلمات المرور متطابقة",
+            "continue_button": " المتابعة",
+            "password_input_error_unknown_error": "خطأ في المصادقة"
+        },
+        "password_verify_page": {
+            "page_title": "تحقق من كلمة المرور | ${business_name}",
+            "heading": "أدخل كلمة المرور الخاصة بك",
+            "description": "أدخل كلمة المرور الخاصة بك الخاصة ب ${email_address}",
+            "password_input_label": "كلمة المرور",
+            "password_input_error_required": "الرجاء إدخال كلمة المرور",
+            "password_input_error_incorrect": "الرجاء إدخال كلمة مرور صحيحة",
+            "password_input_error_too_short": "يجب أن تتكون كلمات المرور من 8 أحرف على الأقل",
+            "continue_button": "المتابعة ",
+            "cant_sign_in_description": "لا تستطيع تسجيل الدخول ؟",
+            "cant_sign_in_link": "إعادة تعيين كلمة المرور ",
+            "password_input_error_unknown_error": "خطأ في المصادقة"
+        },
+        "account_locked_page": {
+            "page_title": "الحساب مقفل | ${business_name}",
+            "heading": "الحساب مقفل",
+            "description": "لقد حاولت تسجيل الدخول عدة مرات وتم قفل حسابك لمدة ${lockout Dur Minions} دقيقة",
```

Comment:
@onderay lockout Dur Minions ;), should be `${lockout_duration_minutes}`

Reply:
@evgenyk updated